calculator.rb

```diff
@@ -0,0 +1,57 @@
+# Welcome message for user
+puts "Welcome to the Ada Calculator!"
+puts "These are the following choices:"
+puts "1. add(+)"
+puts "2. subtract(-)"
+puts "3. multiply(*)"
+puts "4. divide(/)"
+puts "Please choose an operand:"
+
+# Accepting user choice (case insensitive)
+operand = gets.chomp.downcase
+
+# Ensuring user is choosing a valid operation
+until ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"].include? (operand) do
+  puts "Please enter a valid operand"
+  operand = gets.chomp.downcase
+end
+
+# Asking user to input values to be used in calculations
+puts "Please enter the first value: "
+first_value = gets.chomp.to_f
+
+# Insert logic to ensure first_value entered is a number
+
+puts "Please enter the second value: "
+second_value = gets.chomp.to_f
+
+# Insert logic to ensure second_value entered is a number
+
+# Defining methods for each operand
+def addittion(x, y)
+  return (x + y)
+end
+
+def subtraction(x, y)
+  return (x - y).to_f
+end
+
+def multiplication(x, y)
+  return (x * y).to_f
+end
+
+def division (x , y)
```

Comment:
Even though the code still runs, most style guides, style conventions, and text editors like Atom will complain/give a warning if there's a space between the method name and the parentheses. aka:

```ruby
def division (x , y)
```
should be this:
```ruby
def division(x , y)
```